firmware/papracode/papracode.ino

```diff
@@ -110,11 +120,11 @@ void setup() {
   pinMode(led2, OUTPUT);
   pinMode(led3, OUTPUT);
   pinMode(led4, OUTPUT);
-  pinMode(PWMPin, OUTPUT);  //PA3 - TCA0 WO3, pin1 on 14-pin parts
-  TCA0.SPLIT.CTRLB=TCA_SPLIT_LCMP0EN_bm|TCA_SPLIT_HCMP0EN_bm; //PWM on WO0 & WO3
-  TCA0.SPLIT.LPER=0xFF; // Count down from 255 on WO0/WO1/WO2
-  TCA0.SPLIT.HPER=0xFF; // Count down from 255 on WO3/WO4/WO5
-  TCA0.SPLIT.CTRLA=TCA_SPLIT_CLKSEL_DIV4_gc|TCA_SPLIT_ENABLE_bm; //enable the timer with prescaler of 4
+  pinMode(PWMPin, OUTPUT);  //PB0 - TCA0 WO0, pin9 on 14-pin parts
```

Comment:
Will we have to go back to SPLIT if and when the buzzer is implemented?

Reply:
Good catch, let me look